src/main/java/com/codingnomads/betty/logic/services/GameInformationService.java

```diff
@@ -13,42 +15,45 @@
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
 @Service
 public class GameInformationService {
```

Comment:
This class is getting very crowded. You don't have to address it right now since we have a clean up coming up but it might be good looking at it. It is massive right now, almost 200 lines of code